hub/util/from_tfds.py

```diff
@@ -0,0 +1,43 @@
+import tensorflow  # type: ignore
+import tensorflow_datasets as tfds  # type: ignore
+from tqdm import tqdm  # type: ignore
+
+from hub import Dataset
+
+
+def from_tfds_to_path(
+    tfds_dataset_name: str, split: str, hub_ds_path: str, batch_size: int = 100
+):
+    """Converts the tfds dataset with name `tfds_dataset_name` into a hub dataset and saves it at `hub_ds_path`
+    Args:
+        tfds_dataset_name (str): Name of tfds dataset.You can see a list of all tfds datasets here:
+            https://www.tensorflow.org/datasets/catalog/overview
+        split (str) : Enum for dataset splits used for loading a certain tfds dataset split
+        hub_ds_path (str): Path where new hub dataset will be created
+        batch_size (int): Batch size for tfds dataset
```

Comment:
Is this purely a performance parameter or does this have an effect on the output? If it does, please test it.

Reply:
```suggestion
        batch_size (int): Batch size for tfds dataset. Has no effect on output, but may affect performance.
```